src/euphorie/client/model.py

```diff
@@ -1486,6 +1507,77 @@ def in_place_custom_measures(self):
         return self.measures_of_type("in_place_custom")
 
 
+class Choice(SurveyTreeItem):
+    """User choice."""
+
+    __tablename__ = "choice"
+    __mapper_args__ = dict(polymorphic_identity="choice")
+
+    id = schema.Column(
+        "id",
+        types.Integer(),
+        schema.ForeignKey(SurveyTreeItem.id, onupdate="CASCADE", ondelete="CASCADE"),
+        primary_key=True,
+    )
+    condition = schema.Column(types.String(512), nullable=True)
+
+    @property
+    def is_visible(self):
+        if not self.condition:
+            return True
+        options = self.condition.split("|")
```

Comment:
Apparently it is visible only if we have some paths.
What about calling this column `allowed_paths`?

Reply:
I don't understand. Which column? Why rename it?

Reply:
No, sorry, that doesn't fit at all. It's a condition. If any of the options that correspond to the paths is picked, then the choice is visible.